auth_backends/backends.py

```diff
@@ -34,6 +30,22 @@ class EdXOpenIdConnect(OpenIdConnectAuth):
 
     auth_complete_signal = django.dispatch.Signal(providing_args=["user", "id_token"])
 
+    @property
+    def ID_TOKEN_ISSUER(self):
+        return self.setting('ISSUER')
```

Comment:
Note that the consuming service will need to have this value set (in addition to URL_ROOT). This value should be set in configuration.